runtime/rsyslog.h

```diff
@@ -673,7 +685,7 @@ struct actWrkrIParams {
 #  define  __attribute__(x)  /*NOTHING*/
 #endif
 
-#ifndef O_CLOEXEC
+#if !defined(O_CLOEXEC) && !defined(_AIX)
```

Comment:
so this works on AIX even if the define does not exist? Sounds strange, pls double-check.

Reply:
You are certainly right. I think the problem was due to includes order in source code as rsyslog.h was included just after config.h and before system's ones. So as soon as the include order is adjusted in all source code file the prepocessor test would be rolled back.

Reply:
I guess we still need to fix this one here.